nativelib/src/main/scala/scala/scalanative/meta/LinktimeInfo.scala

```diff
@@ -8,9 +8,15 @@ import scala.scalanative.unsafe._
 object LinktimeInfo {
   @resolvedAtLinktime("scala.scalanative.meta.linktimeinfo.isWindows")
   def isWindows: Boolean = resolved
+  
+  @resolvedAtLinktime("scala.scalanative.meta.linktimeinfo.isMac")
+  def isMac: Boolean = resolved
 
   @resolvedAtLinktime(
     "scala.scalanative.meta.linktimeinfo.isWeakReferenceSupported"
   )
   def isWeakReferenceSupported: Boolean = resolved
+  
+  @resolvedAtLinktime("scala.scalanative.meta.linktimeinfo.isArm64")
+  def isArm64: Boolean = resolved
```

Comment:
Now, after merging https://github.com/scala-native/scala-native/pull/2452 we can detect CPU architecture at runtime in a convenient way in the same way as on the JVM. As long as we don't use any Arm64 specific functions we should use runtime resolved value instead. 
I would suggest moving this method to `scala.scalanative.runtime.PlatformExt` (`Platform` is extern module and it can only contain extern methods, so we can define runtime resolved `object PlatformExt`) where we would use the same logic as on the JVM to check if we are using Arm64 based machine.